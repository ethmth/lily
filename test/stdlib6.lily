
let a::int = [3, 2, 1]

print(len(a))
print(truelen(a))

a.append(4)

print(len(a))
print(truelen(a))
printl(a)


