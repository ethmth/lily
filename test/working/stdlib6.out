3
3
4
7
[3, 2, 1, 4]
